python/ray/tune/search/nevergrad/nevergrad_search.py

```diff
@@ -166,6 +170,13 @@ def __init__(
                     "pass a list of parameter names or None as the `space` "
                     "parameter."
                 )
+            if self._optimizer_kwargs:
+                raise ValueError(
```

Comment:
Minor wording nit:

```python
raise ValueError(
    "If you pass in optimizer kwargs, either pass "
    "an `Optimizer` subclass or an instance of "
    "`ConfiguredOptimizer`."
)
```

Reply:
Changed the error message you suggested. thanks.